main.tf

```diff
@@ -1,18 +1,40 @@
 module "tags" {
   source = "git::https://github.com/rhythmictech/terraform-terraform-tags.git?ref=v0.0.2"
   tags   = var.tags
-
```

Comment:
Don't remove this line. Our standards are to keep an empty line above any multiline argument.

Reply:
Added the space again. I didn't realize I had removed it.